### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hab6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,13 +139,15 @@ export ODBCINI=$II_SYSTEM/ingres/files/odbc.ini
...
+#import ingres_sa_dialect
 
 print('Python %s on %s' % (sys.version, sys.platform))
 print('SQLAlchemy %r' % sqlalchemy.__version__)
 con_str = 'ingres:///demodb'  # local demodb
 #con_str = 'ingres://dbuser:PASSWORD@HOSTNAME:27832/db'  # remote database called "db"
 print(con_str)
-print(ingres_sa_dialect.base.dialect().create_connect_args(url=sqlalchemy.engine.make_url(con_str)))
+# If the next line is uncommented, need to also uncomment: import ingres_sa_dialect
+#print(ingres_sa_dialect.base.dialect().create_connect_args(url=sqlalchemy.engine.make_url(con_str)))
```
Comment (clach04) [inline README.md:150]: alternative is to import and print on the same line with a semi-colon which would make this cleaner (whilst still being dirty ;-))

**Comment by clach04** [comment]
Looks  good, thanks!

File: README.md [hunk 2]
```diff
@@ -99,12 +100,12 @@ Ingres dialect tested with pyodbc and pypyodbc (pypyodbc useful for debugging, t
 
 Download Ingres dialect for SQLAlchemy:
 
-    git clone https://github.com/clach04/ingres_sa_dialect.git
+    git clone -b rel_1_4_42 https://github.com/clach04/ingres_sa_dialect.git
```
Comment (clach04) [inline README.md:103]: Is this actually needed? I thought master supported both (but it's been a while so I'm hazy without running this)

Comment (hab6) [reply README.md:103]: Cloning the [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy) repository without specifying any branch currently pulls version `2.0.0b2` from default branch `main`, which if used, causes `sqlalchemy.create_engine()` to fail, whereas with SQLAlchemy version `1.4.42`, it succeeds.

### Test Script: `1test.py`
```
(py3_sa) λ cat 1test.py
import sys
import sqlalchemy

print('Python %s on %s' % (sys.version, sys.platform))
print('SQLAlchemy %r' % sqlalchemy.__version__)
con_str = 'ingres:///demodb'  # local demodb
#con_str = 'ingres://dbuser:PASSWORD@HOSTNAME:27832/db'  # remote database called "db"
print(con_str)
#print(ingres_sa_dialect.base.dialect().create_connect_args(url=sqlalchemy.engine.make_url(con_str)))

engine = sqlalchemy.create_engine(con_str)
connection = engine.connect()
query = 'SELECT * FROM iidbconstants'
for row in connection.execute(sqlalchemy.text(query)):
        print(row)
```

### FAILURE using SQLAlchemy version `2.0.0b2`
```
(py3_sa) λ pip list | grep sqlalchemy
ingres-sa-dialect 0.3          c:\users\mhabiger\data\test\sqlalchemy\ingres_sa_dialect\lib
SQLAlchemy        2.0.0b2.dev0 c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib
```
```
(py3_sa) λ python 1test.py 
Python 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)] on win32 
SQLAlchemy '2.0.0b2'  
ingres:///demodb 
C:\Users\mhabiger\Data\pvm\py3_sa\test\1test.py:11: SADeprecationWarning: The dbapi() classmethod on dialect classes has been renamed to import_dbapi().
  Implement an import_dbapi() classmethod directly on class <class 'ingres_sa_dialect.pyodbc.Ingres_pyodbc'> to remove this warning; the old .dbapi() cl
assmethod may be maintained for backwards compatibility. 
  engine = sqlalchemy.create_engine(con_str)   
Traceback (most recent call last):   
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\base.py", line 1388, in execute 
    meth = statement._execute_on_connection    
AttributeError: 'str' object has no attribute '_execute_on_connection'  
  
The above exception was the direct cause of the following exception:    
  
Traceback (most recent call last):   
  File "C:\Users\mhabiger\Data\pvm\py3_sa\test\1test.py", line 12, in <module>    
    connection = engine.connect()    
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\base.py", line 3210, in connect 
    return self._connection_cls(self)
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\base.py", line 147, in __init__ 
    self._dbapi_connection = engine.raw_connection()
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\base.py", line 3234, in raw_connection    
    return self.pool.connect()  
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\base.py", line 397, in connect    
    return _ConnectionFairy._checkout(self)    
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\base.py", line 1193, in _checkout 
    fairy = _ConnectionRecord.checkout(pool)   
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\base.py", line 660, in checkout   
    rec = pool._do_get()   
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\impl.py", line 168, in _do_get    
    with util.safe_reraise():   
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\util\langhelpers.py", line 148, in __exit__ 
    raise exc_value.with_traceback(exc_tb)
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\impl.py", line 166, in _do_get    
    return self._create_connection() 
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\base.py", line 338, in _create_connection   
    return _ConnectionRecord(self)   
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\base.py", line 623, in __init__   
    self.__connect()  
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\pool\base.py", line 856, in __connect  
    )._exec_w_sync_on_first_run(self.dbapi_connection, self)  
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\event\attr.py", line 473, in _exec_w_sync_on_first_run
    self(*args, **kw) 
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\event\attr.py", line 487, in __call__  
    fn(*args, **kw)   
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\util\langhelpers.py", line 1868, in go 
    return once_fn(*arg, **kw)  
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\create.py", line 746, in first_connect    
    dialect.initialize(c)  
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\default.py", line 480, in initialize 
    self.default_schema_name = self._get_default_schema_name( 
  File "c:\users\mhabiger\data\test\sqlalchemy\ingres_sa_dialect\lib\ingres_sa_dialect\base.py", line 693, in get_default_schema_name    
    rs = connection.execute(sqltext) 
  File "c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib\sqlalchemy\engine\base.py", line 1390, in execute 
    raise exc.ObjectNotExecutableError(statement) from err    
sqlalchemy.exc.ObjectNotExecutableError: Not an executable object: "SELECT dbmsinfo('username')" 
```

### SUCCESS using SQLAlchemy version `1.4.42`
```
(py3_sa) λ pip list | grep sqlalchemy
ingres-sa-dialect 0.3         c:\users\mhabiger\data\test\sqlalchemy\ingres_sa_dialect\lib
SQLAlchemy        1.4.42.dev0 c:\users\mhabiger\data\pvm\py3_sa\test\sqlalchemy\lib
```
```
(py3_sa) λ python 1test.py
Python 3.10.7 (tags/v3.10.7:6cc6b13, Sep  5 2022, 14:08:36) [MSC v.1933 64 bit (AMD64)] on win32
SQLAlchemy '1.4.42'
ingres:///demodb
('mhabiger                        ', 'mhabiger                        ', '$ingres                         ')
```

### ToDo
As I understand, a main goal is to get the Ingres dialect working with SQLAlchemy version 2 (in beta for now).

Comment (clach04) [reply README.md:103]: > Cloning the [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy) repository without specifying any branch currently pulls version `2.0.0b2` from default branch `main`, which if used, causes `sqlalchemy.create_engine()` to fail, whereas with SQLAlchemy version `1.4.42`, it succeeds.
> .... snip

But that clone isn't the SQLAlchemy repo, its this (dialect) repo. This dialect repo has no 2.x support

Comment (hab6) [reply README.md:103]: Oh, you are right! My bad. I'll create another PR to fix the readme. Thanks.

**Pull request merged**